packages/form-field/src/FormFieldContext/FormFieldContext.tsx

```diff
@@ -10,6 +10,7 @@ export interface FormFieldContextProps {
   size: Size;
   state: FormFieldState;
   inputProps?: FormFieldInputElementProps;
+  optional?: boolean;
```

Comment:
Do we want to make this a prop, or leave it as `contentEnd`?

Reply:
Talked offline and I think we're on the same page